src/components/ProviderInitializer/helpers/getIframeProvider.ts

```diff
@@ -1,11 +1,14 @@
 import { IframeProvider } from '@multiversx/sdk-web-wallet-iframe-provider/out';
+import { IframeLoginTypes } from '@multiversx/sdk-web-wallet-iframe-provider/out/constants';
 
 export async function getIframeProvider({
   address,
-  walletUrl
+  walletUrl,
+  loginType
 }: {
   address: string;
   walletUrl: string;
+  loginType?: IframeLoginTypes;
```

Comment:
This should be an interface instead of having the types declared like this.

Reply:
`IframeLoginTypes` it's enum